packages/fxa-settings/src/components/Settings/ProductPromo/index.tsx

```diff
@@ -24,24 +24,51 @@ export interface ProductPromoProps {
 export const ProductPromo = ({
   type = ProductPromoType.Sidebar,
 }: ProductPromoProps) => {
-  const { attachedClients } = useAccount();
+  const { attachedClients, subscriptions } = useAccount();
 
   const hasMonitor = attachedClients.some(
     ({ name }) => name === MozServices.Monitor
   );
-  // const hasMonitorPlus = subscriptions.some(
-  //   ({ productName }) => productName === MozServices.MonitorPlus
-  // );
+  const hasMonitorPlus = subscriptions.some(
+    ({ productName }) => productName === MozServices.MonitorPlus
+  );
```

Comment:
I was hoping this would be pretty easy after that initial set up 🎉  I guess we'll want to at least double check Monitor Plus does actually come through with this name. I wasn't sure if we wanted to use Client IDs for both of these, it would be nice if we could just go ahead and do FXA-8533 at some point...

Reply:
I don't think we can use `clientId` here since it is the same for Monitor and Monitor Plus? I confirmed with clewis that this was the name of the subscription. I agree it is a bit fragile though.